src/input/input.module.css

```diff
@@ -80,7 +84,36 @@
 }
 
 /* Chrome, Firefox, Opera, Safari 10.1+ */
+.input-group ::-webkit-input-placeholder {
+	font-family: 'Montserrat';
+	color: #262626;
+	opacity: 1;
+	font-weight: 400;
+	font-size: 14px;
+}
+.input-group ::-moz-placeholder {
+	font-family: 'Montserrat';
+	color: #262626;
+	opacity: 1;
+	font-weight: 400;
+	font-size: 14px;
+}
+.input-group :-ms-input-placeholder {
+	font-family: 'Montserrat';
+	color: #262626;
+	opacity: 1;
+	font-weight: 400;
+	font-size: 14px;
+}
+.input-group ::-ms-input-placeholder {
+	font-family: 'Montserrat';
+	color: #262626;
+	opacity: 1;
+	font-weight: 400;
+	font-size: 14px;
+}
```

Comment:
If this is not already added automatically by our setup, we should implement that instead of duplicating it here.

Reply:
(Especially because this code already exists verbatim one line lower 😛)

Reply:
Legit I tried grouping and nesting them but the autoprefixer keeps breaking them apart again, so I just ended up putting it back as it was because changing it made it messier 😅